src/Nethermind/Nethermind.Api/Extensions/IConsensusWrapperPlugin.cs

```diff
@@ -20,9 +20,10 @@
 
 namespace Nethermind.Api.Extensions
 {
-    public interface IConsensusWrapperPlugin : INethermindPlugin
+    public interface IConsensusWrapperPlugin : IConsensusPlugin
     {
-        Task<IBlockProducer> InitBlockProducer(IConsensusPlugin consensusPlugin);
+        void WrapPlugin(IConsensusPlugin consensusPlugin);
```

Comment:
QQ: maybe we should have new interface: IBlockProductionPlugin with method InitBlockProducer and later IConsensusWrapperPlugin : IBlockProductionPlugin
IConsensusPlugin : IBlockProductionPlugin

WrapPlugin -> WrapBlockProductionPlugin(IBlockProductionPlugin plugin) 

WDYT?